Cargo.toml

```diff
@@ -14,19 +14,26 @@ homepage    = "https://github.com/pimeys/a2"
 documentation = "https://docs.rs/a2"
 edition = "2018"
 
+[features]
+default = ["openssl"]
+ring = ["dep:ring", "pem"]
```

Comment:
maybe it actually makes sence to remove this feature and use the negative feature selector `#[cfg(not(feature = "openssl"))]`. Then dependency on `ring` becomes an impl detail and we don't have to require MSRV 1.60.

Reply:
Actually this is not possible. We need a dedicated feature flag to enable crates.

Reply:
https://github.com/rust-lang/cargo/issues/1839